### ISSUE TITLE
Mask RCNN implementations - additional info

### ISSUE THREAD

**Issue by IgorKasianenko** [RedditSota/state-of-the-art-result-for-machine-learning-problems#42]
Hi, I'd like to add several models implementing Mask R-CNN. 
First one is Facebook [Detectron](https://github.com/facebookresearch/Detectron) in Caffe2. Works good.
Another one is in [Tensorflow](https://github.com/CharlesShang/FastMaskRCNN) with custom Slim library. This one is not supported by author, but works.  
Last one is [MXNet](https://github.com/TuSimple/mx-maskrcnn)

About the [link ](https://github.com/matterport/Mask_RCNN)in the description - it is Keras on top of TensorFlow, not pure TensorFlow.
Hope it helps. 

P.S. May you add guidance in what format people should add pull requests?

**Comment by taoyudong** [comment]
I have updated the information. Thanks for your suggestion. The original author has not been active for a while. I will try to update the page as all of you suggest.

For pull request, I would prepare a template and post it in the repo. Thanks for your suggestion again!
